waves-integration/src/main/scala/com/wavesplatform/dex/grpc/integration/clients/combined/CombinedWavesBlockchainClient.scala

```diff
@@ -45,7 +45,8 @@ class CombinedWavesBlockchainClient(
   settings: Settings,
   matcherPublicKey: PublicKey,
   meClient: MatcherExtensionClient,
-  bClient: BlockchainUpdatesClient
+  bClient: BlockchainUpdatesClient,
+  mkCombinedStream: => CombinedStream
```

Comment:
why by name?

Reply:
It preserves the old behavior, when we initialize CombinedStream later. I think we can change it to call by value.

Reply:
Fixed